src/tools/rbd/action/Bench.cc

```diff
@@ -63,14 +81,18 @@ struct rbd_bencher {
   Mutex lock;
   Cond cond;
   int in_flight;
+  enum io_type io_type;
+  uint64_t io_size;
 
-  explicit rbd_bencher(librbd::Image *i)
+  explicit rbd_bencher(librbd::Image *i, enum io_type io_type, uint64_t io_size)
```

Comment:
It is because type name matches variable name. Change type name to `io_type_t`? Or use CamelCase.